examples/browser/test/pages/main-page.ts

```diff
@@ -62,10 +62,49 @@ export class MainPage {
     }
 
     public getxBarTabPosition(tabNumber: Number): Number {
-        return browser.getLocation(`ul.p-MenuBar-content > .p-MenuBar-item:nth-child(${tabNumber}`, 'x')
+        return browser.getLocation(`ul.p-MenuBar-content > .p-MenuBar-item:nth-child(${tabNumber}`, 'x');
     }
 
     public getxMenuPosition(): Number {
-        return this.driver.getLocation('.p-Widget.p-Menu.p-MenuBar-menu', 'x')
+        return this.driver.getLocation('.p-Widget.p-Menu.p-MenuBar-menu', 'x');
     }
+
+    public expansionFolder(): Boolean {
```

Comment:
Boolean checks should start with is/has/was or something similar so that's it's a bit easier to read i.e isExpansionFolderAvailable (just an example, I'm not sure what it is exactly).